src/adapter/src/coord/appends.rs

```diff
@@ -158,10 +172,16 @@ impl Coordinator {
     /// immediately. Otherwise we must wait for `now()` to advance past the timestamp chosen for the
     /// writes.
     #[tracing::instrument(level = "debug", skip(self))]
-    pub(crate) async fn try_group_commit(&mut self) {
+    pub(crate) async fn try_group_commit(&mut self, permit: Option<GroupCommitPermit>) {
         let timestamp = self.peek_local_write_ts();
         let now = Timestamp::from((self.catalog().config().now)());
-        if timestamp > now {
+
+        // We want to allow system writes that have external requests waiting on them, through as
+        // quickly as possible.
```

Comment:
I'm not entirely convinced about this, why would this not be true for user writes?

Reply:
Updated to call out that this is a hack to get tests passing